### PR TITLE
Add doctest to dates

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: doc/src/manual/dates.md [hunk 1]
```diff
@@ -293,20 +296,47 @@ julia> Dates.dayofquarter(t)
 ```
 
 The [`dayname()`](@ref) and [`monthname()`](@ref) methods can also take an optional `locale` keyword
-that can be used to return the name of the day or month of the year for other languages/locales:
+that can be used to return the name of the day or month of the year for other languages/locales.
+There are also versions of these functions returning the abbreviated names, namely
+[`dayabbr()`](@ref) and [`monthabbr()`](@ref).
+Firs the mapping is loaded into the `LOCALES` variable:
```
Comment (tkelman) [inline doc/src/manual/dates.md]: First

**Review by kshyatt — APPROVED** [review]
Assuming that typo gets fixed, LGTM. Thanks!

File: doc/src/manual/dates.md [hunk 2]
```diff
@@ -181,7 +184,7 @@ julia> dt2 = DateTime(2000,2,1)
 2000-02-01T00:00:00
 
 julia> dt - dt2
-381110402000 milliseconds
+381110400000 milliseconds
```
Comment (quinnj) [inline doc/src/manual/dates.md]: My guess is the original result came from when Dates was accounting for leap seconds by default (back before it was in Base).

Comment (tkelman) [inline doc/src/manual/dates.md]: checks out - there was 1 in 1998, 1 in 2005, 1 in 2008, and 1 in June 2012, so 2 in that interval

**Pull request merged**